helm-charts/basehub/templates/dex/configmap.yaml

```diff
@@ -0,0 +1,67 @@
+{{- if .Values.dex.enabled -}}
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: dex
+  labels:
+    app: dex
+data:
+  dex.yaml: |
+    issuer: https://{{ .Values.dex.hubHostName }}/services/dex
+    storage:
+      type: sqlite3
+      config:
+        # /srv/db is a PVC mounted for persistence
+        file: /srv/db/dex.sqlite
+
+    web:
+      # Listen on all interfaces, so this is publicly visible
+      http: 0.0.0.0:5556
+
+    oauth2:
+      # Don't explicitly require users to grant access via the
+      # dex interface, for a smoother experience
+      skipApprovalScreen: true
+
+    connectors:
+    - type: oauth
+      id: hub
+      name: hub
+      config:
+        clientID: service-dex
+        # Env vars are expanded via gomplate, which is present in the
+        # upstream dex docker image
+        clientSecret:  {{ "{{" }} .Env.HUB_OAUTH2_CLIENT_SECRET {{ "}}" }}
+        redirectURI: https://{{ .Values.dex.hubHostName }}/services/dex/callback
+        userIDKey: name
+        tokenURL: http://proxy-public/hub/api/oauth2/token
+        authorizationURL: https://{{ .Values.dex.hubHostName }}/hub/api/oauth2/authorize
+        userInfoURL: http://proxy-public/hub/api/user
+
+    staticClients:
+    - id: oauth2-proxy
+      redirectURIs:
+      -  https://{{ .Values.dex.hubHostName }}/services/oauth2-proxy/oauth2/callback
+      name: oauth2-proxy
+      # Env vars are expanded via gomplate, which is present in the
+      # upstream dex docker image
+      secret: {{ "{{" }} .Env.OAUTH2_PROXY_CLIENT_SECRET {{ "}}" }}
```

Comment:
Breaks common YAML practice of indententing list entries, valid though so just formatting aspect.